primitives/src/bazaar.rs

```diff
@@ -57,11 +57,11 @@ pub type OfferingIdentifier = u32;
 #[cfg_attr(feature = "serde_derive", derive(Serialize, Deserialize))]
 #[cfg_attr(feature = "serde_derive", serde(rename_all = "camelCase"))]
 pub struct OfferingData {
-	pub url: PalletString,
+	pub url: UnboundedPalletString,
```

Comment:
Could it be that you mixed up `PalletString` and `UnboundedPalletString` in the whole bazaar domain?

Reply:
oh, right. no i didnt mix this up, but this stems from the fact that i did not do any migrations yet for the bazaar pallet. honestly i dont remember why. let's discuss in the meeting if we will continue using the bazaar pallet in the current form and if yes, open a new issue for this. what do you think?

Reply:
Yes, let's discuss that 👍